packages/manager/modules/pci/src/projects/project/project.html

```diff
@@ -84,22 +84,40 @@
                                         </a>
                                     </div>
                                 </div>
-                                <h2
-                                    data-translate="pci_projects_project_documentation"
-                                ></h2>
-                                <ul>
-                                    <li
-                                        data-ng-repeat="link in $ctrl.links track by $index"
+                                <div class="row">
+                                    <div
+                                        class="col-sm-6 col-lg-6 col-xl-6 mb-4"
```

Comment:
No `col-md-6` ? Also, you need to specify for all the sizes ?

Reply:
I think it will be the same as col-lg-6. Correct me if I am wrong, if the lower-end classes are not provided, it will consider the next higher class. For example, if only col-xl-6 is provided, bootstrap will consider 6 column layout for the rest of the lower-end classes. In this case, providing col-xl-6 is sufficient.

Reply:
Yes you're right, so no need `col-sm/lg-6` ? :)

Reply:
We are using bootstrap4 in this module (https://github.com/ovh/manager/blob/master/packages/manager/modules/pci/package.json#L67).

If I check the documentation, I see the following:

"_Grid breakpoints are based on minimum width media queries, meaning they apply to that one breakpoint and all those above it (e.g., .col-sm-4 applies to small, medium, large, and extra large devices, but not the first xs breakpoint)._"

So, it should be the contrary no ? (lower classes applies to higher sizes)

Reply:
Yes, you are right, it's exactly the opposite of what I said :). In this case, just having col-sm-4 is sufficient.

Reply:
You mean `col-sm-6` (not `col-sm-4`) is required? Isn't it?